src/test/java/io/vertx/core/eventbus/ClusteredEventBusTest.java

```diff
@@ -516,18 +516,12 @@ public void testCustomDeliveryStrategy() throws Exception {
   }
 
   private static class CustomDeliveryStrategy implements DeliveryStrategy {
-    private VertxInternal vertx;
     private ClusterManager clusterManager;
     private String rack;
 
     @Override
     public void setVertx(Vertx vertx) {
-      this.vertx = (VertxInternal) vertx;
-    }
-
-    @Override
-    public void eventBusStarted() {
-      clusterManager = vertx.getClusterManager();
+      clusterManager = ((VertxInternal) vertx).getClusterManager();
```

Comment:
I actually meant that the delivery strategy has an init method passing Vertx and the ClusterManager which avoid the delivery strategy go pull the cluster manager as it is provided in init method.